### ISSUE TITLE
train script?

### ISSUE THREAD

**Issue by dourcer** [anguyen8/deepface-emd#2]
Good job. I read the paper and code carefully, but I didn't find the training script. Can you provide some training script examples? thank you.

**Comment by dourcer** [comment]
finetuning ArcFace

**Comment by haithanhp** [comment]
You can use [arcface-pytorch](https://github.com/ronghuaiyang/arcface-pytorch) to train masked faces.

**Comment by anguyen8** [comment]
@dourcer Thanks for your question! 

We will be sharing the training scripts in this repo within a week or so.

**Comment by haithanhp** [comment]
Hi @dourcer ,

The training scripts with datasets for the training are available [here](https://github.com/anguyen8/deepface-emd/tree/main/arcface-pytorch). Let us know if there is anything.
